packages/web/components/Landing/sections/Build.tsx

```diff
@@ -4,7 +4,7 @@ import BackgroundImage from 'assets/landing/build-background.png';
 export const Build: React.FC = () => (
   <Box
     width="100%"
-    minHeight="100%"
+   //minHeight="100%" 
```

Comment:
This fails because the linter requires a space after the `//`. I'm surprised that you didn't get an error when you ran `git push`.

Just remove the line & it'll be fine.

```suggestion
```

Reply:
@dysbulic done the changes!

Reply:
You just removed the `//` & put the bug back in.

Reply:
Hey @dysbulic , Added solution check out and let me know if that works. I am having another file if this doesn't work.

Reply:
@Anosh21 did you test the solution? your solution was supposed to fix the landing page, which currently looks like this: https://the-game-git-fork-anosh21-develop-metafam.vercel.app/landing

Reply:
Ah I see, I need to merge the other file , this file has some conflicts, As it was not pulled from my fork though,
I am closing the request, Will open new one @vidvidvid .